src/parser/tptp/Tptp.g

```diff
@@ -698,17 +698,17 @@ definedFun[CVC4::ParseOp& p]
                   MK_TERM(api::GT, decPart, SOLVER->mkReal(1, 2)),
                   // if decPart > 0.5, round up
                   MK_TERM(api::TO_INTEGER,
-                          MK_TERM(api::PLUS, n, SOLVER->mkReal(1))),
+                          MK_TERM(api::PLUS, n, SOLVER->mkInteger(1))),
```

Comment:
Is it necessary that this change happens?

Reply:
ditto

Reply:
Same issue: performance (change) vs intuitiveness (original).